### DISCUSSION TITLE
Admin password cannot be reset

### DISCUSSION THREAD

**Discussion by Kurayami-Senpai** [requarks/wiki#6762] [Help / Questions] [answered]
I had the problem that I couldn't log in as an admin, so I wanted to reset my password. However, when I entered the correct email, I received an error stating that the email type was Invalid. Now I was lucky and I remembered my password, but I would have had a big problem if not because I couldn't reset my password. I noticed that it changes as soon as I add a .com instead of a .de to the email, but my email is a .de

Can this be fixed or is it simply not possible to reset an admin password?

**Comment by fnafox** [comment, marked as answer]
Forgotten admin password can be reset by [editing the database entry](https://docs.requarks.io/en/troubleshooting#how-to-manually-reset-the-admin-password).

**Reply by cesarnr21** [reply]
I have the same issue where I am not able to log it. I followed those instructions and when I ran `docker exec db psql -U wiki -d wiki -c "UPDATE users SET password = 'HASH-PASSWORD' WHERE email = 'YOUR-EMAIL';"` (with my email and password-hash), I received some ouput `UPDATE 1`, so it seems like it like it is working, but I am still not able to log in.

**Reply by fnafox** [reply]
did you make sure to hash the password before running the SQL query?

**Reply by cesarnr21** [reply]
Yes, used <https://bcrypt-generator.com/> like the instructions recommended, I entered a password and made sure it had 12 rounds, then `encrypt`. Then used that output for the SQL query.

**Reply by cesarnr21** [reply]
To give you more information, I got logged out after I stopped the wiki js intance from running on the default port 80, then I restarted it on another port using nginx reverse proxy. When I go to wiki js on the new port, I can only see the homepage but none of the other pages I wrote which is why I need to login.

I also tried changing the port back 80 but I am still logged out.

**Reply by fnafox** [reply]
The password reset procedure should work if your instance is configured correctly and connecting to the right database. Whether you access it directly or through a reverse proxy should have no effect on this. Can you log in with a different user, are you sure you're using the local authentication strategy?

**Reply by cesarnr21** [reply]
I did not do any of the set up at **Authentication** -> **Add Strategy**, I don't remeber seeing that on the Getting Started documentation. It's fine, I tried for a few hours and decided to just create a new instance of wikijs and then restored my backup so I didn't lose anything.

I will review the documentation again and set up anything that would allow me to reset the password in case I forget it again.

Thank you for your help.
